fetch/content-type/resources/script-content-types.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "input": ["text/javascript;charset=windows-1252"],
+    "executes": true,
+    "encoding": "windows-1252"
+  },
+  {
+    "input": ["text/javascript;\";charset=windows-1252"],
+    "executes": true,
+    "encoding": "windows-1252"
+  },
+  {
+    "input": ["text/javascript\u000C"],
+    "executes": false,
+    "encoding": null
+  },
+  {
+    "input": ["\"text/javascript\""],
+    "executes": false,
+    "encoding": null
+  },
+  {
+    "input": ["text / javascript"],
+    "executes": false,
+    "encoding": null
+  },
+  {
+    "input": ["x/x", "text/javascript"],
+    "executes": true,
+    "encoding": null
+  },
+  {
+    "input": ["x/x;charset=windows-1252", "text/javascript"],
+    "executes": true,
+    "encoding": null
+  },
+  {
+    "input": ["text/javascript", "x/x"],
+    "executes": false,
+    "encoding": null
+  },
+  {
+    "input": ["text/javascript; charset=windows-1252", "text/javascript"],
+    "executes": true,
+    "encoding": "windows-1252"
+  },
+  {
+    "input": ["text/javascript;\"", "x/x"],
+    "executes": true,
+    "encoding": null
+  },
+  {
+    "input": ["text/javascript", ""],
+    "executes": true,
+    "encoding": null
+  },
+  {
+    "input": ["text/javascript", "error"],
+    "executes": true,
+    "encoding": null
+  },
+  {
+    "input": ["text/javascript;charset=windows-1252", "x/x", "text/javascript"],
```

Comment:
Should the first one also be of a type that doesn't execute, for this test and the next 3 as well?

Reply:
Then the encoding wouldn't copy over.